src/current/_includes/releases/v24.1/v24.1.0-alpha.4.md

```diff
@@ -41,8 +41,7 @@ Release Date: March 25, 2024
 
 <h3 id="v24-1-0-alpha-4-operational-changes">Operational changes</h3>
 
-- The [cluster setting](../v24.1/cluster-settings.html) `admission.wal.failover.unlimited_tokens.enabled` can be set to `true` to cause unlimited admission tokens during write-ahead log (WAL) failover. This should not be changed without significant expertise, since the default, which preserves the token counts from the preceding non-WAL-failover interval, is expected to be safer. [#120135][#120135]
-- The new [`cockroach start`](../v24.1/cockroach-start.html) option `--wal-failover=among-stores` or `COCKROACH_WAL_FAILOVER=among-stores` environment variable will configure a multi-store CockroachDB node to fail over a store's write-ahead log (WAL) to another store's data directory. Failing over the write-ahead log may allow some operations against a store to continue to complete even if the underlying storage is temporarily unavailable. [#120509][#120509]
+- The new [`cockroach start`](../v24.1/cockroach-start.html) option [`--wal-failover=among-stores` or `COCKROACH_WAL_FAILOVER=among-stores`]({% link v24.1/cockroach-start.md %}#write-ahead-log-wal-failover) environment variable will configure a multi-store CockroachDB node to fail over a store's write-ahead log (WAL) to another store's data directory. Failing over the write-ahead log may allow some operations against a store to continue to complete even if the underlying storage is temporarily unavailable. [#120509][#120509]
```

Comment:
We should note that this feature is in `PREVIEW`

Reply:
Updated to say it's in preview